.github/workflows/check.yaml

```diff
@@ -0,0 +1,124 @@
+#
+# CI for this project needs to do two things:
+#   1. Setup the environment for and run `yarn build`
+#   2. Build the distribution rpm for use in QE/OST/Integration testing
+#
+name: run CI on PRs
+on:
+  pull_request:
+
+jobs:
+  test_el8_offline:
+    name: EL8 - test the PR (ovirt-engine-nodejs-modules build)
+
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/centos/centos:stream8
```

Comment:
please use quay.io/ovirt/buildcontainer:el8stream (https://github.com/oVirt/buildcontainer), it has all deps installed already

Reply:
and you can drop all the additional dnf installs below.